package.json

```diff
@@ -164,7 +164,6 @@
         "nx": "17.0.0",
         "nyc": "^15.1.0",
         "postcss-loader": "^3.0.0",
-        "prettier": "^1.19.1",
```

Comment:
Wouldn't it be better to keep prettier and eslint configs at the root level?

Reply:
Will add it back later. Planning to send PRs as chunks package-by-package :)